pkg/operation/botanist/component/vpa/templates/crd-verticalpodautoscalers.tpl.yaml

```diff
@@ -1,7 +1,9 @@
 ---
-# For backwards compatibility it's not possible to take the community based VPA definition since it tightens
-# the validation too much. Instead, the CRD needs to retain and permit unknown fields as it used to be with
-# the `v1beta1` CRD version.
+# Source: https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml
+# For backwards compatibility, the following customizations were performed on the original CRD manifest:
+# - allow nullable status for v1beta2, v1
+# - additional fields for v1beta2:
+#   - .spec.resourcePolicy.containerPolicies[].controlledResources
```

Comment:
Can you elaborate on these customizations? Why we had to do them? Is it something that we customize with this PR or is it already the case in the master (and we only document this now)?

Reply:
Those were in the version I was reverting to, which had some - now unnecessary - Gardener-specific patches. Per offline discussions, I have modified this PR to update the VPA-related CRDs to version "0.11.0", instead of revert to an earlier version.